app/src/main/java/com/pvp/app/ui/screen/settings/Settings.kt

```diff
@@ -78,6 +85,35 @@ fun CategoryRow(
     )
 }
 
+@Composable
+fun ReminderSetting() {
+    Row {
+        Text(
+            text = "Reminder time",
+            style = MaterialTheme.typography.titleLarge,
+        )
+    }
+
+    Text(
+        modifier = Modifier.padding(vertical = 8.dp),
+        text = "Selected (minutes): $reminderTimeMinutes",
```

Comment:
Minutes selected, but what does it mean? What does `Reminder time` setting do? Does it remind me something every `X` minutes? Does it remind me something every `X` minutes before/after task? Or is it something else? User cannot known as it is now and won't be able to use it correctly from the first try.